audb/core/load_to.py

```diff
@@ -324,12 +325,13 @@ def load_to(
     # to signal all files were correctly loaded
 
     _save_database(db, db_root, db_root_tmp, num_workers, verbose)
+    os.remove(database_lock_path(db_root_tmp))
```

Comment:
Why do we have to remove the lock file here manually?

Reply:
Because there is a test afterwards to make sure that `db_root_tmp` is empty.

Reply:
And `audb.load()` does not create a `,lock` file int he tmp folder so we don't have to remove it?
Why is it needed for `load_to()` and why is `load_to()` not locking the `root` folder where the data will be stored in the end?
I think we also need this as two users might call `audb.load_to()` at the same time using the same `root` folder as target.

Reply:
As far as I remember `load_to()` is not working on the audb cache, except for getting the dependency file. So don't think we need to lock here. That it still creates a lock file in the temporary cache folder is just a side effect of calling `load()`.

> I think we also need this as two users might call audb.load_to() at the same time using the same root folder as target.

Puh, really?

Reply:
I think it might indeed not be needed as the process is not safe anyway.
Even if the loading to that folder is locked user2 might change the content before user1 publishes a new version from that folder.

Reply:
Could you then use `_load_header()` instead of `load_header()` in `load_to()` to avoid creating the `.lock` file in the first place?

Reply:
Good idea, I changed it accordingly and rename `_load_header()` to `load_header_wo_lock()`.